src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.util.ArrayList;
+
+import duke.task.Task;
+
+/**
+ * Ui is a class that prints messages to the user.
+ */
+public class Ui {
+    /**
+     * Prints a welcome message.
+     */
+    public static void welcomeMessage() {
+        String catLogo = "        /\\_____/\\\n" +
+                "       /  o   o  \\\n" +
+                "      ( ==  ^  == )\n" +
+                "       )         (\n" +
+                "      (           )\n" +
+                "     ( (  )   (  ) )\n" +
+                "    (__(__)___(__)__)";
+        System.out.println(catLogo);
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println("    Hello! I'm NEKOBOT!!");
+        System.out.println("    What can I do for you :>");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Prints a goodbye message.
+     */
+    public static void goodbyeMessage() {
+        System.out.println("    ____________________________________________________________");
+        System.out.println("     Bye~ Hope to see you again soon ;w;");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    /**
+     * Prints all tasks in the TaskList.
+     * @param taskList An ArrayList containing all existing Tasks.
+     */
+    public static void listMessage(ArrayList<Task> taskList) {
```

Comment:
Can consider changing the naming to printTasks(...) so as to make it a verb (ie an action)